src/Varnish.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\Varnish\Varnish.
+ */
+
+namespace Netzstrategen\Varnish;
+
+/**
+ * Varnish client functionality.
+ */
+class Varnish {
+
+  /**
+   * URLs to purge in Varnish whose values are Booleans indicating regex.
```

Comment:
Probably state it's an associative array whose keys are the URLs and the values are the Booleans indicating regex.